docs/Programmers_Guide.rst

```diff
@@ -89,8 +89,8 @@ Test code for testing the scale contraction functionality and log metrics for F6
 Infrastructure
 ^^^^^^^^^^^^^^
 
-- CMake is used to build and package hipTensor. There are :code:`CMakeLists.txt` files throughout the code.
-- `Doxygen/Breathe/Sphinx/ReadtheDocs` are used to produce documentation. Content for the documentation is from:
+- CMake is used to build and package hipTensor. There are ``CMakeLists.txt`` files throughout the code.
```

Comment:
I am not sure, my changes here were valid. Please check in the built documentation.

Reply:
This line is still valid, but it does nothing. I think it does no harm, but if there is a need for new commits I'll change it back to make the code more consistent.